eng/tools/XplatPackageSigner/XplatPackageSigner.proj

```diff
@@ -0,0 +1,38 @@
+<!--
+  Code signing of .nupkg's built on Linux/macOS cannot be code-signed on the same machine which built the package.
+  This project takes as inputs a folder of *.nupkg packages and code signs them using MicroBuild.
+
+  Note: because Authenticode signing of .dll's is not something Linux and macOS can verify anyways, this signing
+  process only code-signs the .nupkg itself, not the contents.
+-->
+<Project DefaultTargets="AfterBuild" InitialTargets="CheckForRequiredProperties">
+  <PropertyGroup>
+    <SignType>$([MSBuild]::ValueOrDefault($(SignType),'real'))</SignType>
+  </PropertyGroup>
+
+  <Import Project="..\..\..\Directory.Build.props" />
```

Comment:
To confirm: MSBuild doesn't automatically import `Directory.Build.*` files for *.proj files? And, you don't want to import `$(RepositoryRoot)\build\sources.props` anyhow?

Reply:
correct. Don't need sources.props since I don't need to run restore on this project.